### PR TITLE
Create ecps-profiles.php

### PR DESCRIPTION
**Pull request by cdavis-ubc**

### CODE REVIEW

File: templates/education/ecps-profiles.php [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+            <?php if ( false !== $last_name ) : echo esc_html( $last_name ); endif; ?>
+            <?php if ( false !== $position ) : echo esc_html( $position ); endif; ?> 
+            <?php if ( false !== $clone_scholarly_interests ) : echo esc_html( $clone_scholarly_interests ); endif; ?> 
+            <?php if ( false !== $research_interests ) : echo esc_html( $research_interests ); endif; ?> 
+            <?php echo esc_html( $plain_term_name ); ?>
+          </div>
+
+          <?php if ( false !== $clone_room_number ) : echo esc_html( $clone_room_number ); endif; ?></br>
+
+          <?php if ( false !== $email ) :  echo wp_kses_post( "\n"."<i class='icon-envelope'></i> <a href='mailto:$email'>$email</a>"."\n" ) ;  endif; ?></a></br>
```
Comment (richardtape) [inline templates/education/ecps-profiles.php]: This isn't quite right. If the $email is false, you'll still get the closing `</a>` tag.

File: templates/education/ecps-profiles.php [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+          <?php if ( false !== $phone1 ) : echo wp_kses_post( "<i class='icon-phone-sign'></i>"." (".$phone1.") ".$phone2."-".$phone3 ); endif; ?></br>
```
Comment (richardtape) [inline templates/education/ecps-profiles.php]: Perhaps a better way to do the logic/markup here (and for several other places) is;

```
<?php if ( false !== $phone1 ) :  ?>
    <i class='icon-phone-sign'></i><?php echo esc_html( $phone1 ) . " " . esc_html( $phone2 ) . "-" . esc_html( $phone3 ); ?>
<?php endif; ?>
```

This way we have less markup being output by PHP and it'll be easier to read in the template.

**Comment by cdavis-ubc** [comment]
Ok thanks for the notes, very helpful!
I have added the improved markup to ecps-profiles.php
Once I have it right with this one I will edit the remaining profile templates and send pull requests.

**Comment by richardtape** [comment]
OK this looks good!

**Comment by richardtape** [comment]
One thing though, let's test this on verf before you send any more through. I'll get it up there now

**Comment by richardtape** [comment]
OK, it's updated on Verf, @cdavis-ubc - let me know

**Comment by cdavis-ubc** [comment]
Ok will test and get back to you.

**Pull request merged**